docs/docs/visual-testing-with-storybook.md

```diff
@@ -99,6 +99,29 @@ module.exports = ({ config }) => {
 
 > Note that if you're using a [StaticQuery](/docs/static-query/) in your components, `babel-plugin-remove-graphql-queries` is required to render them in Storybook. This is because the queries are run at build time in Gatsby, and will not have been run when rendering the components directly.
 
+> For typescript projet, add this rule:
+
+```js:title=.storybook/webpack.config.js
+...
+
+config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    loader: require.resolve('babel-loader'),
+    options: {
+      presets: [['react-app', {flow: false, typescript: true}]],
+      plugins: [
+        require.resolve('@babel/plugin-proposal-class-properties'),
```

Comment:
Just wondering if we need `@babel/plugin-proposal-class-properties`. Doesn't typescript has class properties support by default?

Reply:
i tried without this plugins and i have a error "Support for the experimental syntax 'classProperties' isn't currently enabled".
It's required.

Reply:
👍  thanks for validating.